src/prommis/uky/tests/test_uky_flowsheet.py

```diff
@@ -547,23 +547,23 @@ def test_solution(system_frame):
     ].value == pytest.approx(2.6984e-4, 1e-4)
 
     assert model.fs.roaster.gas_outlet.flow_mol[0].value == pytest.approx(
-        0.0085064, 1e-4
```

Comment:
Are these changes expected? They seem significant.

Reply:
I definitely expected some changes, but perhaps @jinliangma can speak as to whether the magnitude of the changes is within reason or not

Reply:
This is my only remaining concern, everything else looks good to me.

Reply:
@bpaul4 These changes are expected because the amount of liquid (moisture) entering the roaster has changed significantly, and the latest commit exacerbates this change even more.